### ISSUE TITLE
ReferenceError: slug is not defined

### ISSUE THREAD

**Issue by clochix** [cozy-labs/konnectors#234]
Seen in the logs of a user: application crash when starting:
```
/usr/local/cozy/apps/konnectors/build/server/init/konnectors.js:82
    log.info("Reseting isImporting field for " + slug + "...");
                                                 ^

ReferenceError: slug is not defined
    at konnectorResetValue (/usr/local/cozy/apps/konnectors/build/server/init/konnectors.js:82:50)
    at /usr/local/cozy/apps/konnectors/build/server/init/konnectors.js:61:18
    at /usr/local/cozy/apps/konnectors/node_modules/async/lib/async.js:181:20
    at Immediate.iterate [as _onImmediate] (/usr/local/cozy/apps/konnectors/node_modules/async/lib/async.js:262:13)
    at processImmediate [as _immediateCallback] (timers.js:383:17)
```

**Comment by clochix** [comment]
We should use konnector.slug in https://github.com/cozy-labs/konnectors/blob/master/server/init/konnectors.coffee#L72

**Comment by clochix** [comment]
Fixed in 7e64e38, thanjs @frankrousseau !
